mojo/stdlib/stdlib/collections/optional.mojo

```diff
@@ -40,6 +40,10 @@ from utils import Variant
 # TODO(27780): NoneType can't currently conform to traits
 @fieldwise_init
 struct _NoneType(ImplicitlyCopyable, Movable):
+    alias __copyinit__is_trivial = True
+    alias __moveinit__is_trivial = True
+    alias __del__is_trivial = True
```

Comment:
For `NoneType` these aliases are already synthesized by the compiler so we shouldn't need to manually add them here.

Reply:
Oh odd I was thinking of the stdlib `NoneType` - I'm not sure of what the purpose of this type is. 
If we can just remove the `__copyinit__` function then the compiler should do everything correctly 👍